lib/ext/adviser.rb

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+module AdviserExtension
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def to_csv(_ = {})
+      CSV.generate do |csv|
+        csv << ['Ref. Number', 'Name', 'Firm']
+
+        find_each do |adviser|
+          csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        end
+      end
+    end
+  end
+end
+
+Adviser.send(:include, AdviserExtension)
```

Comment:
Is there a reason to add the include outside of the Adviser class definitions?
Other thing, I don't want to over engineering, but what happens if we need another type of csv for the Adviser model? Since it was added the to_csv to the Adviser model on this extension, the scope of the to_csv, *it's closed for extension*. Can we not use a wrapper on Adviser model instead a module?